src/containers/OrderUpdate.vue

```diff
@@ -75,6 +79,90 @@ export default {
       } catch (err) {
         this.handleError(err);
       }
+    },
+
+    onPrint() {
+      if (!this.params) return [];
+
+      const { categories, squares, parts } = this.params;
+
+      var dd = {
+        content: [
+          {
+            stack: [
+              this.$t("score"),
+              {
+                text: `${this.order.date}`,
+                style: "subheader"
+              }
+            ],
+            style: "header"
+          },
+          {
+            text: [
+              `${this.$t("clientName")}: ${this.order.clientName}\n`,
+              `${this.$t("phoneNumber")}: ${this.order.phoneNumber}\n`,
+              `${this.$t("carModel")}: ${this.order.make}\n`,
```

Comment:
why?) make and model - different things, make it's make

Reply:
maker maybe

Reply:
OK